packages/web_framework/HISTORY.rst

```diff
@@ -3,9 +3,9 @@ History
 
 .. to_doc
 
--------
-23.2rc1
--------
+---------------------
```

Comment:
These should match the number of characters in `23.2.dev0`. How did `23.2rc1` get created in the first place ?

Reply:
I'll fix the number of characters. The previous version `23.2rc1` originated here 64a813e763d23ac8725edffde04b95478a9d9e2a